setup.py

```diff
@@ -5,7 +5,7 @@
 with open('README.rst', 'r') as readme:
     README_TEXT = readme.read()
 
-VERSION = '0.2.1.dev0'
+VERSION = '0.5.dev0'
```

Comment:
I would prefer jump to 0.5.
If I remember correctly we agreed (or at least I understood) that it was desirable that both wrappers and core maintain the same 0.X version.